crates/cargo-contract/src/cmd/extrinsics/instantiate.rs

```diff
@@ -467,6 +467,11 @@ impl InstantiateDryRunResult {
             DEFAULT_KEY_COL_WIDTH
         );
         name_value_println!("Data", format!("{:?}", self.data), DEFAULT_KEY_COL_WIDTH);
+        name_value_println!(
+            "Gas limit",
```

Comment:
```suggestion
            "Gas consumed",
```
It is true that this value is used for the gas limit for transactions. But it is still the `Gas consumed` by the dry-run. Does it fit with the max col width?